Update and Copy landnode.ks to landlongnode.ks.

MarkAndJester opened the pull request:
I tried to use landnode.ks to land at a particular longitude every time (instead of in relation to where the sun was), but the new periapsis kept moving.
I realized the Mun was rotating whilst the ship cruised up to the node and also whilst the ship descended to the new periapsis after the node was created.
ah ha! Lines 51 through 60 make the corrections! I have left the original sun-angle code in. One could write some soft of if-then based on the parameters input so this code could do either "land before sunset" or "land at specified longitude"... p.s. thank you for all this code. It is wonderful.

MarkAndJester committed:
Update and Copy landnode.ks to landlongnode.ks.

I tried to use landnode.ks to land at a particular longitude every time (instead of in relation to where the sun was), but the new periapsis kept moving.
I realized the Mun was rotating whilst the ship cruised up to the node and also whilst the ship descended to the new periapsis after the node was created.
ah ha! Lines 51 through 60 make the corrections! I have left the original sun-angle code in. One could write some soft of if-then based on the parameters input so this code could do either "land before sunset" or "land at specified longitude"... p.s. thank you for all this code. It is wonderful.

baloan commented:
Hi, thanks for your contribution. I'd like to add it to the codebase. Is there any way to reopen the pull request?

MarkAndJester commented:
Re-opening pull request. Sorry about that. I got excited writing my first code for kOS and tried to use this GitHub system and messed it up. I've got a couple of changes to this though. I added sin() to take into account latitude being different to 0deg; I have an orbit change script I think would be good to add to your collection, so this extra sin() will come in handy.

MarkAndJester commented on script/landlongnode.ks:
```diff
@@ -35,3 +48,13 @@ print "T+" + round(missiontime) + " Descent orbit burn: " + round(vom) + ", dv:"
 
+// adjust for the body we are orbiting to rotate under us whilst we cruise to the descending node.
+set shiptonodedegrees to 360 * etam / body:rotationperiod.
+set adjustforshiptonodeseconds to ops * shiptonodedegrees / 360.
```
should now read
set adjustforshiptonodeseconds to sin(90 - ABS(ORBIT:INCLINATION)) * ops * shiptonodedegrees / 360.

MarkAndJester commented on script/landlongnode.ks:
```diff
@@ -35,3 +48,13 @@ print "T+" + round(missiontime) + " Descent orbit burn: " + round(vom) + ", dv:"
+set adjustforshiptonodeseconds to ops * shiptonodedegrees / 360.
+// adjust for the body we are orbiting to rotate during the descent to periapsis to land with "run landv(3)." after the node is created.
+set adjustfororbitdegrees to 360 * (nd:orbit:period / 2) / body:rotationperiod.	
+set adjustfororbitseconds to ops * adjustfororbitdegrees / 360.
```
should now read
set adjustfororbitseconds to sin(90 - ABS(nd:ORBIT:INCLINATION)) * ops * adjustfororbitdegrees / 360.

MarkAndJester committed:
Update landlongnode.ks

Added sin() to adjustforshiptonodeseconds and adjustfororbitseconds to allow for latitude being different to 0deg.
